src/test/resources/getRemoteConfig.json

```diff
@@ -50,7 +50,7 @@
       "name": "dev-admin",
       "imageUrl": "http://image.jpg"
     },
-    "updateTime": "Wed, 30 Sep 2020 17:56:07 GMT",
+    "updateTime": "2020-11-03T20:24:15.203Z",
```

Comment:
This is not quite the format described in the comment in the source (it had 9 digit nano second value).

Reply:
Good catch!